great_expectations/experimental/metric_repository/metrics.py

```diff
@@ -76,19 +169,30 @@ class ColumnMetric(Metric, Generic[_ValueType]):
 # bring together a domain type, value type and any parameters (aka metric_value_kwargs)
 
 # TODO: Add metrics here for all Column Descriptive Metrics
-#  QuantileValuesColumnMetric is an example of a metric that has parameters
+#  ColumnQuantileValuesMetric is an example of a metric that has parameters
 
 
-class QuantileValuesColumnMetric(ColumnMetric[List[float]]):
+class ColumnQuantileValuesMetric(ColumnMetric[List[float]]):
     quantiles: List[float] = Field(description="Quantiles to compute")
-    allow_relative_error: Union[str, float] = Field(
+    allow_relative_error: Union[float, str] = Field(
         description="Relative error interpolation type (pandas) or limit (e.g. spark) depending on data source"
     )
 
+    @property
+    @override
+    def value_type(self) -> str:
+        return "list[float]"
+
+    @property
+    def metric_type(self) -> str:
+        return self.__class__.__name__
+
 
 class MetricRun(MetricRepositoryBaseModel):
     """Collection of Metric objects produced during the same execution run."""
 
-    id: uuid.UUID = Field(description="Run id")
```

Comment:
I removed the id fields on MetricRun and Metric since they will be generated by the backend and don't yet have a use here in OSS. If in the future we retrieve metrics with IDs we can add this back in as an optional field.